parcel.go

```diff
@@ -22,8 +23,8 @@ func (s ParcelStore) Add(p Parcel) (int, error) {
 		sql.Named("address", p.Address),
 		sql.Named("created_at", p.CreatedAt))
 	if err != nil {
-		fmt.Println("Error during 'Add' operation:", err)
-		return 0, nil
+		fmt.Println("Error during 'Add' operation:")
```

Comment:
Так в выводе можно оставить, она тут не мешает (ну или тогда весь вывод убрать)
```suggestion
		fmt.Println("Error during 'Add' operation:", err)
```